doc/api/async_context.md

```diff
@@ -147,6 +147,49 @@ this time will print the stack trace and exit. See
 Creating an async resource within the `onPropagate` callback will result in
 a recursive call to `onPropagate`.
 
+### `asyncLocalStorage.bind(fn [, thisArg])`
```

Comment:
Nit: I would remove the `thisArg` param. The only time I've ever found it useful was for binding to `this` when passing as a callback, and that's easily achieved with arrow functions now.

Reply:
That's fair

Reply:
It _can_ be helpful for patterns like:

```js
class Thing {
  constructor() {
    this.method = AsyncLocalStorage.bind(this.method, this)
  }

  method() {
    // ...
  }
}
```

Also, shouldn't this be upper-cased since it's a static?

```suggestion
### `AsyncLocalStorage.bind(fn [, thisArg])`
```

Reply:
> shouldn't this be upper-cased since it's a static?
Oh yeah, that and being prefixed with `Static method: `. Doing now for this and the snapshot method